src/Chapter2.hs

```diff
@@ -621,7 +641,12 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven[x] = [x]
+takeEven [x, y] = [x]
+takeEven (x:y:xs) = x : takeEven xs
```

Comment:
Great!
As `y` is not used in the right part, you can use wildcard instead of the name `y`:

```suggestion
takeEven (x:_:xs) = x : takeEven xs
```